scripts/linters/python_linter.py

```diff
@@ -320,6 +341,35 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from pylint error messages.
+
+        Args:
+            lint_messages: list(str). Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # Remove newlines and coverage report from the end of message.
+        # we need to remove last five items from the list becuase starting from
+        # end we have three lines with just newline characters, fourth line
+        # contains the coverage report i.e.
+        # Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
+        # and one line with dashes(---).
+        lint_messages = lint_messages[:-5]
+        for message in lint_messages:
+            # Every pylint message has a message id inside the brackets
+            # so we need to check if it is true and if case is true then
+            # remove the message-id from the end of original message.
+            if message.endswith(')'):
+                last_string_length = len(message.split()[-1])
```

Comment:
What's last_string? I don't know what that means semantically. If it's the error code, then say so.

Better still, use a regex to get rid of it. That way, you can verify the format of what you're removing. (The reason I keep asking you to do this is so that we can be sure that you haven't missed any cases in your analysis. If anything errors out, devs will start telling you fairly quickly.)